CommandLine.c

```diff
@@ -398,8 +398,6 @@ int CommandLine_run(int argc, char** argv) {
    Machine_scan(host);
    Machine_scanTables(host);
    CommandLine_delay(host, 75);
-   Machine_scan(host);
-   Machine_scanTables(host);
 
```

Comment:
Pretty sure you can drop the delay on line 400 if we go down this path...?  I've always wondered why this second scan/scanTable was there in the original code - perhaps to guarantee values are fully setup (two samples mean all counters are rate converted and ready to display when the UI is presented).  But, I'm all for this change if it works.

Reply:
If really needed, could the delay perhaps be reduced from 75 to 1 (ms)?

Reply:
A reduction should be fine.